README.md

```diff
@@ -41,37 +41,36 @@ more information about password hash : https://linuxconfig.org/how-to-hash-passw
 
 more info: https://fai-project.org/fai-guide
 
-**installing a debug image:**
+**Classes and Flags Customization:**
 
-A debug image with more debug packages installed is available through the grub menu.\
-All it does is add "dbg" to the list of FAI_FLAGS.
+Running ./build_debian_iso/build_iso.sh with the "--custom" option will allow a user:
```

Comment:
Add a line to mention that running `./build_debian_iso.sh` without custom flag will do :  all classes downloaded, and only one grub option with clustermode/lvmraid/english/no_debug/no_kerberos/no_cockpit

Reply:
will do